### PR TITLE
[MIG][12.0] account_bank_statement_import_save_file 

### PR DESCRIPTION
**Pull request by alexis-via**
Migrate the module, with README updated to the new subdir system.

### CODE REVIEW

File: account_bank_statement_import_save_file/__init__.py [hunk 1]
```diff
@@ -1,6 +1 @@
-# -*- coding: utf-8 -*-
-# © 2015 Therp BV (<http://therp.nl>).
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (alexey-pelykh) [inline account_bank_statement_import_save_file/__init__.py]: You can keep the license in these files

Comment (alexis-via) [reply account_bank_statement_import_save_file/__init__.py]: No nitpicking please :)

Comment (alexey-pelykh) [reply account_bank_statement_import_save_file/__init__.py]: None intended :) that was just a comment, not a change request ;)

**Review by sebastienbeau — APPROVED** [review]
Code review

**Comment by Creamaster** [comment]
@sebastienbeau Hi, Can this PR be merged ? or is there some missing informations ?

**Pull request merged**